build/build_mock.go

```diff
@@ -1,23 +1,3 @@
-// Copyright (c) 2017 Uber Technologies, Inc.
-//
```

Comment:
- setup precommit hooks to do this for `go` files
- add makefile target to codegen